templates/user/profile.tmpl

```diff
@@ -69,6 +69,17 @@
 								</ul>
 							</li>
 							{{end}}
+							{{if .Badges}}
+							<li>
+								<ul class="user-badges">
+								{{range .Badges}}
+									<li>
+										<img src="{{ .ImageURL }}" alt="{{ .Description}}" />
```

Comment:
Would set a reasonable explicit width/height (probably enforce square) and then add `object-fit: contain` via CSS.